validator/validator.c

```diff
@@ -2397,7 +2418,8 @@ ds_response_to_ke(struct module_qstate* qstate, struct val_qstate* vq,
 		/* errors here pretty much break validation */
 		verbose(VERB_DETAIL, "DS response was error, thus bogus");
 		errinf(qstate, rc);
-		errinf(qstate, "no DS");
+		errinf_ede(qstate, "no DS", reason_bogus);
```

Comment:
It doesn't make sense to use the variable `reason_bogus` here, because it cannot be anything else than `LDNS_EDE_DNSSEC_BOGUS` anyway. But was about `LDNS_EDE_NO_REACHABLE_AUTHORITY` or `LDNS_EDE_NETWORK_ERROR`?

```suggestion
		errinf_ede(qstate, "no DS", LDNS_EDE_NETWORK_ERROR);
```